testing/scenario_app/ios/Scenarios/ScenariosUITests/GoldenImage.m

```diff
@@ -6,6 +6,8 @@
 #import <XCTest/XCTest.h>
 #include <sys/sysctl.h>
 
+static const double kRmseThreshold = 0.5;
```

Comment:
how did you determinate this value?

Reply:
I ran the tests on my machine without changes and that was the smallest threshold that that contains all the errors, plus it is a reasonably low error.

Reply:
what would be an example of a false positive? 

If you change https://github.com/flutter/engine/blob/master/testing/scenario_app/lib/src/platform_view.dart#L523 to `151` or one of the translates by `1` pixel?

Reply:
what would be an example of a false positive?

If you change https://github.com/flutter/engine/blob/master/testing/scenario_app/lib/src/platform_view.dart#L523 to 151 or one of the translates by 1 pixel, does the test pass?

Reply:
It doesn't pass when i change it to 151.  In that case the RMSE goes from 0.49 to 0.94.

Reply:
RMSE factors in how big of a diff there is and is relative to the size of the images.  So small discrepancies on large images make less of a difference.

Reply:
SGTM